File: C#/Usage.cs
using System;
using System.Text;

namespace YIGIM
{
    class Usage
    {
        static void Main(string[] args)
        {
            // Token class initialization with API secret key
            Token token = new Token("A8225D2B5651D9447663A1A21F9430CD");
            // Setting agent name
            token.SetData("Agent007");
            // Calculating timestamp
            TimeSpan span = DateTime.UtcNow -
                new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
            // Token generation
            String result = token.Generate(((long)span.TotalMilliseconds) / 200);
            // Printing result
            Console.WriteLine(result);
        }
    }
}

File: Java/Usage.java
public class Usage {

    public static void main(String[] args)
            throws Exception {
        // Token class initialization with API secret key
        Token token = new Token("08D1230B84123284BD24CAD64B59DF94");
        // Setting agent name
        token.setData("Agent007");
        // Token generation
        String result = token.generate(System.currentTimeMillis()/200);
        // Printing result
        System.out.println(result);
    }
}